css/index.css

```diff
@@ -101,17 +103,106 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes .box{
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
The individual boxes only need to have their color overridden.  They get the rest of the styling from the parent .box.